src/components/Article.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({
+  title,
+  date,
+  text,
+}) => (
```

Comment:
```suggestion
const Article = ({ title, date, text, }) => (
```
Так легче смотрится, по моему мнению, конечно

Reply:
> Так легче смотрится, по моему мнению, конечно

смотрится лучше, но больше 2-х это уже считается много. а значит, как и в HTML, если есть 3-я строка - переношу все строки с новой

Reply:
3 attributes rule usually is applied only to HTML.
In JS you try to make your line no longer than 70-80 chars.